mix.exs

```diff
@@ -39,7 +39,7 @@ defmodule TelemetryMetricsStatsd.MixProject do
   defp deps do
     [
       {:telemetry, "~> 0.4 or ~> 1.0"},
-      {:telemetry_metrics, "~> 0.6"},
+      {:telemetry_metrics, "~> 1.0.0"},
```

Comment:
This is fine, but on other similar deps, we just relax the version requirement with an `or`:

```suggestion
      {:telemetry_metrics, "~> 0.6 or ~> 1.0"},
```